.zenodo.json

```diff
@@ -0,0 +1,27 @@
+{
+    "creators": [
+      {
+        "affiliation": "Goddard Space Flight Center",
+        "name": "Jeff Klenzing",
+        "orcid": "0000-0001-8321-6074"
+      }, 
+      {
+        "affiliation": "U.S. Naval Research Laboratory", 
+        "name": "Angeline G. Burrell",
+        "orcid": "0000-0001-8875-9326"
+      }, 
```

Comment:
Does the order matter here?  Should we swap these moving forward?

Reply:
Order does matter here, I suppose we should swap.